src/api.js

```diff
@@ -70,7 +69,7 @@ export default class Api {
      * The even emitter instance.
      * @type {EventEmitter}
      */
-    this.events = options.events || new EventEmitter();
+    this.events = options.events;
```

Comment:
Personally, I'd prefer the API change since it'd be more obvious it's expected. I don't mind either way, though. Let me know what you prefer and I'll do it. /cc @michielbdejong